quill-cassandra/src/main/scala/io/getquill/sources/cassandra/CqlIdiom.scala

```diff
@@ -22,7 +22,10 @@ object CqlIdiom {
       case a: Function  => a.body.show
       case Infix(parts, params) =>
         StringContext(parts: _*).s(params.map(_.show): _*)
-      case a @ (_: Function | _: FunctionApply | _: Dynamic | _: If | _: OptionOperation | _: Query) =>
+      case a @ (
+        _: Function | _: FunctionApply | _: Dynamic | _: If | _: OptionOperation |
+        _: Query | _: Block | _: Val
```

Comment:
@jilen good catch, but that's from before this PR, should probably handle separately.